core/jazz_events-handler/test/test.js

```diff
@@ -1,10 +1,263 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+'use strict';
```

Comment:
not needed

Reply:
why?

Reply:
use strict is only needed when u r using const and let(and other things as well) on node4.3.
We are running on 6.10 and will be running on 8.10 so there is no need for doing use strict.

Reply:
ok